.github/workflows/xml-schema-validation.yml

```diff
@@ -0,0 +1,37 @@
+name: XML Schema Validation
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  manifest:
+    name: Manifest
+    uses: CruGlobal/mobile-content-api/.github/workflows/validate-schema.yml@ghaXmlSchemaValidation
```

Comment:
I need to change this to reference `master` once this PR actually gets merged.

If I change it before I merge the PR, then it won't be able to find the referenced workflow